.gitmodules

```diff
@@ -39,7 +39,9 @@
 	url = https://github.com/Azure/sonic-platform-common
 [submodule "src/sonic-platform-daemons"]
 	path = src/sonic-platform-daemons
-	url = https://github.com/Azure/sonic-platform-daemons
+	url = https://github.com/MaratGubaiev/sonic-platform-daemons
```

Comment:
Please check the changes in this file. I don't think it's related.

Reply:
You need to update the submodule with Azure/sonic-platform-daemons once the changes in sonic-platform-daemon gets merged.